### PR TITLE
fix: rework the validation checking and tidy up

### PR DESCRIPTION
**Pull request by gashcrumb**
fixes [ENTESB-13842](https://issues.redhat.com/browse/ENTESB-13842)

### CODE REVIEW

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx [hunk 1]
```diff
@@ -257,52 +249,11 @@ export const ApiConnectorCreatorSelectMethod: React.FunctionComponent<IApiConnec
...
-            </Split>
-          </StackItem>
-        </>
-      )}
       <StackItem>
         <ButtonLink
           id={'button-next'}
           data-testid={'button-next'}
-          disabled={!valid}
+          disabled={isDisabled()}
```
[the remark this answers is not in the document]
Comment (kahboom) [reply app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx:256]: Actually just seeing you're not passing this to another component or function.

**Comment by kahboom** [comment]
retest this please

**Comment by gashcrumb** [comment]
retest this please

**Comment by avano** [comment]
retest this please

**Comment by avano** [comment]
retest this please

**Pull request merged**